### DISCUSSION TITLE
About git fork repository cannot sync in my account!

### DISCUSSION THREAD

**Discussion by tomapull** [EddieHubCommunity/support#6849] [Q&A]
Made own fork repository from the organization's repository.
But cannot sync between own and the organization.
Yes How about do it.

**Comment by SohamRoy-01** [comment]
## Hi @tomapull 
* People with write access forked a repository & can sync the fork to the upstream repository. 
* So you can check on GitHub, navigate to the main page of the forked repository that you want to sync with the upstream repository.
* If the changes from the upstream repository cause conflicts, GitHub will prompt you to create a pull request to resolve the conflicts.
* Syncing your fork only updates your local copy of the repository. To update your fork on GitHub.com, you must push your changes
* You can sync a fork branch with the GitHub CLI its a open source tool


# Hope this resolved your problem.
